discord/commands/options.py

```diff
@@ -141,10 +141,8 @@ class Option:
     max_length: Optional[:class:`int`]
         The maximum length of the string that can be entered. Must be between 1 and 6000 (inclusive).
         Only applies to Options with an :attr:`input_type` of :class:`str`.
-    autocomplete: Optional[:class:`Any`]
-        The autocomplete handler for the option. Accepts an iterable of :class:`str` or :class:`OptionChoice`, a callable (sync or async)
-        that takes a single argument of :class:`AutocompleteContext`, or a coroutine.
-        Must resolve to an iterable of :class:`str` or :class:`OptionChoice`.
+    autocomplete: Callable[[:class:`.AutocompleteContext`], Awaitable[Union[Iterable[:class:`.OptionChoice`], Iterable[:class:`str`], Iterable[:class:`int`], Iterable[:class:`float`]]]]
+        The autocomplete handler for the option. Use :func:`discord.utils.basic_autocomplete` to create the callable.
```

Comment:
this seems vague- as if all auto complete handlers have to be basic_autocomplete
maybe something like "basic_autocomplete may also be used ..."

Reply:
True it is a bit vauge. However, I'm not sure how else someone would create an object with: Callable[[:class:`.AutocompleteContext`], Awaitable[Union[Iterable[:class:`.OptionChoice`], Iterable[:class:`str`], Iterable[:class:`int`], Iterable[:class:`float`]]]] manually. Seems a bit tedious without a helper function. Will change though.

Reply:
> True it is a bit vauge. However, I'm not sure how else someone would create an object with: Callable[[:class:`.AutocompleteContext`], Awaitable[Union[Iterable[:class:`.OptionChoice`], Iterable[:class:`str`], Iterable[:class:`int`], Iterable[:class:`float`]]]] manually. Seems a bit tedious without a helper function. Will change though.

You can create said object with
```
async def my_autocomplete_function(ctx):
    return [""]
```